src/i18n/DE/grid-resources.ts

```diff
@@ -0,0 +1,132 @@
+  igx_grid_summary_count: 'Anzahl',
+  igx_grid_summary_min: 'Min',
+  igx_grid_summary_max: 'Max',
+  igx_grid_summary_sum: 'Summe',
+  igx_grid_summary_average: 'Mittel',
+  igx_grid_summary_earliest: 'Früheste',
+  igx_grid_summary_latest: 'Späteste',
+  igx_grid_excel_filter_moving_left: 'Nach links',
+  igx_grid_excel_filter_moving_left_short: 'Links',
+  igx_grid_excel_filter_moving_right: 'Nach rechts',
+  igx_grid_excel_filter_moving_right_short: 'Rechts',
+  igx_grid_excel_filter_moving_header: 'Verschiebe',
+  igx_grid_excel_filter_sorting_asc: 'Aufsteigend',
+  igx_grid_excel_filter_sorting_asc_short: 'Auf',
+  igx_grid_excel_filter_sorting_desc: 'Absteigend',
+  igx_grid_excel_filter_sorting_desc_short: 'Ab',
+  igx_grid_excel_filter_sorting_header: 'Sortiere',
+  igx_grid_excel_filter_clear: 'Lösche Filter',
+  igx_grid_excel_custom_dialog_add: 'Füge Filter hinzu',
+  igx_grid_excel_custom_dialog_clear: 'Lösche Filter',
+  igx_grid_excel_custom_dialog_header: 'Auto-Filter für Spalte: ',
+  igx_grid_excel_cancel: 'Abbrechen',
+  igx_grid_excel_apply: 'Anwenden',
+  igx_grid_excel_search_placeholder: 'Suche',
+  igx_grid_excel_select_all: 'Alles auswählen',
+  igx_grid_excel_select_all_search_results: 'Wähle alle Suchergebnisse aus',
+  igx_grid_excel_add_to_filter: 'Füge aktuelle Auswahl zum Filter hinzu',
+  igx_grid_excel_blanks: '(Leer)',
+  igx_grid_excel_hide: 'Verstecke Spalte',
+  igx_grid_excel_show: 'Zeige Spalte',
+  igx_grid_excel_pin: 'Hefte Spalte an',
+  igx_grid_excel_unpin: 'Löse Spalte',
+  igx_grid_excel_select: 'Selektiere Spalte',
+  igx_grid_excel_text_filter: 'Text-Filter',
+  igx_grid_excel_number_filter: 'Zahlen-Filter',
+  igx_grid_excel_date_filter: 'Datums-Filter',
+  igx_grid_excel_boolean_filter: 'Boolescher-Filter',
+  igx_grid_excel_custom_filter: 'Benutzerdef. Filter...',
+  igx_grid_advanced_filter_title: 'Erweiterte Filterung',
+  igx_grid_advanced_filter_and_group: '"Und" Gruppe',
+  igx_grid_advanced_filter_or_group: '"Oder" Gruppe',
+  igx_grid_advanced_filter_end_group: 'Beende Gruppe',
+  igx_grid_advanced_filter_create_and_group: 'Erstelle "Und" Gruppe',
+  igx_grid_advanced_filter_create_or_group: 'Erstelle "Oder" Gruppe',
+  igx_grid_advanced_filter_and_label: 'und',
+  igx_grid_advanced_filter_or_label: 'oder',
+  igx_grid_advanced_filter_add_condition: 'Bedingung',
+  igx_grid_advanced_filter_ungroup: 'Lösche Gruppierung',
+  igx_grid_advanced_filter_delete: 'Lösche',
+  igx_grid_advanced_filter_delete_filters: 'Lösche Filter',
+  igx_grid_advanced_filter_initial_text: 'Fange an mit einer Gruppe von Bedingungen, die mit "Und" oder "Oder" verknüpft sind',
+  igx_grid_advanced_filter_column_placeholder: 'Wähle Spalte',
+  igx_grid_advanced_filter_value_placeholder: 'Wert',
+  igx_grid_pinned_row_indicator: 'Angeheftet',
+  igx_grid_hiding_check_all_label: 'Verstecke alle',
+  igx_grid_hiding_uncheck_all_label: 'Zeige alle',
+  igx_grid_pinning_check_all_label: 'Hefte alle an',
```

Comment:
I'd follow Excel and use the term "fixieren" instead of "anheften". So: "Alle fixieren"

Reply:
Even though Excel uses the English "freeze" when we built the pinning feature we chose the term for compatibility with the other framework products we have and functionally built it the Excel way with come enhancements. Therefore, I think you are right about following the Excel approach for this action, had it not been the case above, we may have used "freeze" for English as well.

Reply:
Ok, I'll change them all. Note that in Excel they use "fixieren" for pinning/freezing rows/columns. But the pinned files (in the file list) is called "angeheftet".